### DISCUSSION TITLE
cannot delete rollup

### DISCUSSION THREAD

**Discussion by yaoyi** [apache/doris#13901] [RFC]
v1.1.3
`ALTER TABLE table_name DROP ROLLUP rollup_name` cannot delete rollup, it still shows in `SHOW ALTER TABLE ROLLUP from database`

**Comment by LemonLiTree** [comment]
Check whether it is deleted：desc table_name all;
ALTER TABLE table_name DROP ROLLUP rollup_name：This command is an asynchronous operation. It takes a little time to complete
